doc/release/1.4.0-notes.rst

```diff
@@ -24,12 +24,49 @@ For running on PyPy, PyPy3 6.0+ and NumPy 1.15.0 are required.
 Highlights of this release
 --------------------------
 
--
+- a new submodule, `scipy.fft`, now supersedes `scipy.fftpack`; this
+means support for ``long double`` transforms, faster multi-dimensional
+transforms, improved algorithm time complexity, release of the global
+intepreter lock, and control over threading behavior
```

Comment:
I think support for `pydata/sparse` arrays in `scipy.sparse.linalg` can be added here, that's something people will find interesting I think.

Reply:
Good idea, added now, thanks